test/test-utils/dns.js

```diff
@@ -7,8 +7,36 @@ function hexEncodeName(name) {
 function hexEncodeTXT(keys) {
   return '0x' + packet.answer.encode(keys).toString('hex')
 }
+// resolves to a hexadecimal address from a name
```

Comment:
What's the purpose of rewriting this instead of using `dnsName`?

Reply:
It's the same function, I have renamed the "dnsName" function to "nameToHex".  Also, I have moved this "nameToHex" function to the utils section for sharing between the newly created TestOwnedResolver and the old TestPublicResolver.